Changes.md

```diff
@@ -61,6 +61,9 @@ API
   - ShaderView : `registerReferenceScene()` now takes a `filesystem::path`.
   - Catalogue : `save()` and `load()` take a `filesystem::path`, and `generateFileName()` returns one.
   - ExtensionAlgo : `exportExtension()` now takes a `pathlib.Path` in the `directory` argument.
+  - FileSystemPath :
+    - Added a constructor that accepts a `filesystem::path`.
+    - Added a `path()` method returning a `filesystem::path`.
```

Comment:
This needs moving to a new version section, since we've released 1.2.0.0a1 now. Maybe 2023 will be the year when I come up with a better way of managing Changes.md.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5049/commits/0a73bc6a93e4c09b3c2b7add9edb7eff393c92fb